proto/build_event_stream.proto

```diff
@@ -575,6 +575,7 @@ message File {
   int64 length = 6;
 }
 
+// BUILDBUDDY-SPECIFIC EVENTS BELOW
```

Comment:
hm this comment looks like it might've wound up here accidentally? NamedSetOfFiles etc. aren't bb-specific

Reply:
Ah, it should be singular. We added this Tree message